packages/node-cli/src/commands/object/import/ImportObjectsAction.js

```diff
@@ -59,20 +66,29 @@ module.exports = class ImportObjectsAction extends BaseAction {
 
 		try {
 			const flags = [];
+			let scriptIdArray;
 			if (this._runInInteractiveMode) {
 				if (params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS] !== undefined && !params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS]) {
 					flags.push(COMMAND_FLAGS.EXCLUDE_FILES);
 					delete params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS];
 				}
+
+				scriptIdArray = params[ANSWERS_NAMES.SCRIPT_ID].split(' ');
 			} else {
 				this._log.info(NodeTranslationService.getMessage(WARNINGS.OVERRIDE));
 				if (params[COMMAND_FLAGS.EXCLUDE_FILES]) {
 					flags.push(COMMAND_FLAGS.EXCLUDE_FILES);
 					delete params[COMMAND_FLAGS.EXCLUDE_FILES];
 				}
+
+				if (params[ANSWERS_NAMES.OBJECT_TYPE] === IMPORT_OBJECTS_COMMAND_TYPE_PARAM_ALL) {
+					scriptIdArray = (await this._getAllScriptIds(params)).map((el) => el.scriptId).join(' ').split(' ');
+				} if (params[ANSWERS_NAMES.OBJECT_TYPE] !== IMPORT_OBJECTS_COMMAND_TYPE_PARAM_ALL
+					&& params[ANSWERS_NAMES.SCRIPT_ID] === IMPORT_OBJECTS_COMMAND_SCRIPT_ID_PARAM_ALL) {
```

Comment:
I guess it could be an `else if(params[ANSWERS_NAMES.SCRIPT_ID] === IMPORT_OBJECTS_COMMAND_SCRIPT_ID_PARAM_ALL)`

Reply:
Yes you are correct. Updated.

Reply:
@davidecorreu @karl-anthony-ng 
As David said in slack, i think the scriptid here behaves as "contains".
So if a user adds a scriptid which can result in a large number of matching script ID's, this can break the execution as we have now.